### PR TITLE
Adding JS examples, matching the docs

### PR DESCRIPTION
**Pull request by fufu930**
**Thanks for contributing to the Selenium site and documentation!**
**A PR well described will help maintainers to quickly review and merge it**

Before submitting your PR, please check our [contributing](https://selenium.dev/documentation/en/contributing/) guidelines.
Avoid large PRs, help reviewers by making them as simple and short as possible.

### Types of changes

- [ ] Change to the site (I have double-checked the Netlify deployment, and my changes look good)
- [x] Code examples added 
- [ ] Improved translation
- [ ] Added new translation (and I also added a notice to each document missing translation)

### Checklist

- [ ] I have read the [**contributing**](https://selenium.dev/documentation/en/contributing/) document.
- [ ] I have used [hugo](https://gohugo.io/) to render the site/docs locally and I am sure it works.

### CODE REVIEW

File: examples/javascript/test/actionsApi/keysTest.spec.js [hunk 1]
```diff
@@ -1,69 +1,93 @@
...
+                .keyDown(Key.SHIFT)
+                .sendKeys(Key.ARROW_UP)
+                .keyUp(Key.SHIFT)
+                .keyDown(cmdCtrl)
+                .sendKeys('xvv')
+                .keyUp(cmdCtrl)
+                .perform()
+
+            assert.deepStrictEqual(await textField.getAttribute('value'), 'SeleniumSelenium!')
+                // .sendKeys(textField, 'Selenium!') will be possible on next release
```
Comment (harsha509) [inline examples/javascript/test/actionsApi/keysTest.spec.js]: Can you remove the comment in here!

File: examples/javascript/test/actionsApi/keysTest.spec.js [hunk 2]
```diff
@@ -1,69 +1,93 @@
...
+
+            assert.deepStrictEqual(await textField.getAttribute('value'), 'abc')
+        })
+
+        it('Copy and Paste', async function() {
+            await driver.get('https://www.selenium.dev/selenium/web/single_text_input.html')
+
+            var textField = await driver.findElement(By.id('textInput'))
+
+            var cmdCtrl = platform.includes('darwin') ? Key.COMMAND : Key.CONTROL
```
Comment (harsha509) [inline examples/javascript/test/actionsApi/keysTest.spec.js]: const

File: examples/javascript/test/actionsApi/keysTest.spec.js [hunk 3]
```diff
@@ -1,69 +1,93 @@
...
+                .sendKeys(textField, 'abc')
+                .perform()
+
+            assert.deepStrictEqual(await textField.getAttribute('value'), 'abc')
+        })
+
+        it('Copy and Paste', async function() {
+            await driver.get('https://www.selenium.dev/selenium/web/single_text_input.html')
+
+            var textField = await driver.findElement(By.id('textInput'))
```
Comment (harsha509) [inline examples/javascript/test/actionsApi/keysTest.spec.js]: const textField

File: examples/javascript/test/actionsApi/actionsTest.spec.js [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+        })
+
+        after(() => driver.quit())
+
+        it('Pause', async function() {
+            await driver.get('https://selenium.dev/selenium/web/mouse_interaction.html')
+
+            const start = Date.now()
+
+            var clickable = await driver.findElement(By.id('clickable'))
```
Comment (harsha509) [inline examples/javascript/test/actionsApi/actionsTest.spec.js]: const clickable

File: examples/javascript/test/actionsApi/actionsTest.spec.js [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+
+            const end = Date.now() - start
+            assert.ok(end > 2000)
+            assert.ok(end < 4000)
+        })
+
+        it('Clear', async function() {
+            await driver.get('https://selenium.dev/selenium/web/mouse_interaction.html')
+
+            var clickable = driver.findElement(By.id('clickable'))
```
Comment (harsha509) [inline examples/javascript/test/actionsApi/actionsTest.spec.js]: const clickable

File: examples/javascript/test/actionsApi/actionsTest.spec.js [hunk 6]
```diff
@@ -0,0 +1,52 @@
...
+            await driver.actions()
+                .click(clickable)
+                .keyDown(Key.SHIFT)
+                .sendKeys('a')
+                .perform()
+
+            await driver.actions().clear()
+            await driver.actions().sendKeys('a').perform()
+
+            var value = await clickable.getAttribute('value')
```
Comment (harsha509) [inline examples/javascript/test/actionsApi/actionsTest.spec.js]: const value

File: examples/javascript/test/actionsApi/keysTest.spec.js [hunk 7]
```diff
@@ -1,69 +1,93 @@
...
-      const textField = await driver.findElement(By.id("textInput"));
-      await driver.actions()
-        .sendKeys(textField, 'Selenium!')
-        .perform();
-
-      assert.deepStrictEqual('Selenium!', await textField.getAttribute('value'))
-    });
-  });
-});
-
```
Comment (harsha509) [inline examples/javascript/test/actionsApi/keysTest.spec.js:69]: I see here the formatting got changed.
Can you set it to original indent!

**Review by harsha509 — CHANGES_REQUESTED** [review]
Hi @hanceanic ,

Thank you for the PR!

I have requested for few changes, could you please look into those!

**Review by diemol — APPROVED** [review]

**Pull request merged**